hal/src/stm32f2xx/core_hal_stm32f2xx.c

```diff
@@ -345,20 +345,26 @@ void HAL_Core_Config(void)
     PWR_WakeUpPinCmd(DISABLE);
 
     //Wiring pins default to inputs
-#if !defined(USE_SWD_JTAG) && !defined(USE_SWD)
-    for (pin_t pin=0; pin<=19; pin++)
+    for (pin_t pin = 0; pin <= 19; pin++) {
```

Comment:
I understand this is very old code, but since we are modifying it, can we perhaps refactor this somehow, so that we use `TOTAL_PINS`, pin names etc here?